tts_queue_system.py

```
@@ -153,253 +163,262 @@ def estimate_duration(self, text: str) -> float:
             return
-        
+
         self.is_running = True
         self.logger.info("Starting TTS queue system")
-        
+
         # Start worker tasks
         for i in range(self.config.max_concurrent_workers):
             worker = TTSWorker(i, self.config, self.audio_manager)
             self.workers.append(worker)
             asyncio.create_task(self._worker_loop(worker))
```

Ruff (0.15.6)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)